script.js

```diff
@@ -0,0 +1,151 @@
+const form = document.querySelector('form')
+const splash = document.querySelector('.splash')
+const gameBoard = document.querySelector('.game-board')
+
+/** Generates the content for the grid container, with default miss class and unique data-grid-id attributes
+ *
+ * @param height inputted height
+ * @param width inputted width
+ * @returns {string} innerHTML for the grid container
+ */
+ function gridContent(height, width){
+    let innerHTML = ''
+    let gridSize = height * width
+    for (let i = 1; i <= gridSize; i++){
+        let text = `<div class="game-item game-board__sapling miss" data-grid-id="${i}"></div>`
+        innerHTML += text
+
+    }
+    return innerHTML
+}
+
+/** Sets the grid template for the container
+ *
+ * @param height inputted height
+ * @param width inputted width
+ */
+function gridDefinition(height, width){
+    let gridCont = document.querySelector(".game-board__grid")
+    gridCont.innerHTML= gridContent(height, width)
+    gridCont.style.gridTemplateColumns = `repeat(${width}, 1fr)`
+    gridCont.style.gridTemplateRows = `repeat(${height}, 1fr)`
+}
+
+/**
+ * This function changes game-board tiles on click based on hit or miss class
+ *
+ * @param e
+ */
+function hitOrMiss(e) {
+    let target = e.currentTarget.classList
+    if (target.contains('game-board__carrot') ||target.contains('game-board__gopher')) {
+        return
+    } else if (target.contains('miss')) {
+        target.replace('game-board__sapling', 'game-board__carrot')
+        return
+    }
+    target.replace('game-board__sapling', 'game-board__gopher')
+}
+
+/** Checks whether an int is between 3 and 12
+ *
+ * @param int
+ * @returns {boolean}
+ */
+function checkRange(int) {
+    return !(int < 3 || int > 12);
+}
+
+/**
+ * validating form input for row and column numbers
+ *
+ * @returns object
+ */
+function inputGetter() {
+
+    let column = document.querySelector('#column').value
+    let row = document.querySelector('#row').value
+    let gopher = document.querySelector('#gopher').value
+    let parsedRow = parseInt(row)
+    let parsedColumn = parseInt(column)
+    let parsedGopher = parseInt(gopher)
```

Comment:
I'd suggest skipping creating the column/row/gopher variables and instead, parse them directly

let parsedRow = parseInt(document.querySelector('#row').value)

Reply:
Done